source/com.microsoft.tfs.core/rest_core/com/microsoft/teamfoundation/distributedtask/webapi/model/Demand.java

```diff
@@ -10,8 +10,8 @@
 
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.microsoft.tfs.util.StringUtil;
```

Comment:
Are we sure that this change is correct? Is tfs.util.StringUtil.* a perfect match for vss.client.core.utils.StringUtil.*?